internal/server/v1/page/golden/place_page_test.go

```diff
@@ -83,47 +83,47 @@ func TestPlacePage(t *testing.T) {
 				1,
 				[]string{},
 				5,
-				"",
+				"Environment",
 			},
 			{
 				"county.sample.json",
 				"geoId/06085",
 				1,
 				[]string{"Count_HousingUnit_2000To2004DateBuilt"},
 				3,
-				"",
+				"Demographics",
 			},
 			{
 				"state.sample.json",
 				"geoId/06",
 				1,
 				[]string{"Annual_Generation_Electricity"},
 				3,
-				"",
+				"Demographics",
 			},
 			{
 				"city.sample.json",
 				"geoId/0656938",
 				1,
 				[]string{"Median_GrossRent_HousingUnit_WithCashRent_OccupiedHousingUnit_RenterOccupied"},
 				3,
-				"",
+				"Equity",
 			},
 			{
 				"zuid-nederland.sample.json",
 				"nuts/NL4",
 				1,
 				[]string{},
 				5,
-				"",
+				"Overview",
```

Comment:
would be great to have all the categories represented for goldens -- health, equity, economics, ..

Reply:
I will update the golden tests in a follow up PR.